xlive/Blam/Engine/interface/first_person_camera.cpp

```diff
@@ -0,0 +1,84 @@
+#include "stdafx.h"
+#include "first_person_camera.h"
+
+#include "Blam/Cache/TagGroups/unit_definition.hpp"
+#include "Blam/Engine/game/player_control.h"
+#include "Blam/Engine/objects/objects.h"
+#include "Blam/Engine/units/units.h"
+
+#include "H2MOD/Modules/CustomVariantSettings/CustomVariantSettings.h"
+#include "H2MOD/Modules/Shell/Config.h"
+#include "H2MOD/Tags/TagInterface.h"
+#include "Util/Hooks/Hook.h"
+
+#define _USE_MATH_DEFINES
+#include <math.h>
+
+static bool b_showHUD = true;
+
+bool fov_overridden = false;
+float overridden_fov_radians;
+
+
+float observer_suggested_field_of_view()
+{
+	return *Memory::GetAddress<float*>(0x413780, 0x3B5300);
+}
+
+void observer_set_suggested_field_of_view(const float fov)
+{
+	float calculated_radians_FOV;
+	if (currentVariantSettings.forcedFOV == 0)
+	{
+		calculated_radians_FOV = fov * M_PI / 180.0f;
+	}
+	else
+	{
+		calculated_radians_FOV = currentVariantSettings.forcedFOV * M_PI / 180.0f;
+	}
+	WriteValue(Memory::GetAddress(0x413780, 0x3B5300), calculated_radians_FOV);
```

Comment:
use `WriteValue` only when writing to a write protected memory segment/page

Reply:
Changed the following to 
*Memory::GetAddress<float*>(0x413780, 0x3B5300) = calculated_radians_FOV;